API/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
```

Comment:
this shouldn't be here until we start posting

Reply:
Hi August, i keep getting app is not defined if i delete these lines. Am I misunderstanding the question?